server/frontend-server.js

```diff
@@ -0,0 +1,15 @@
+var path = require('path');
+var express = require('express')
+
+module.exports = {
+	app: function() {
+		var app = express();
+		var indexPath = path.join(__dirname, '../dist/index.html');
+		var publicPath = express.static(path.join(__dirname, '../dist/public'));
+
+		app.use('/public', publicPath);
+		app.get('/', function (_, res) { res.sendFile(indexPath) });
```

Comment:
I'd probably still define the first param as `req` or `request` even though we aren't using it here.